### DISCUSSION TITLE
One liner: What is FTD?

### DISCUSSION THREAD

**Discussion by Arpita-Jaiswal** [fastn-stack/fastn#593] [Q&A]
If you have to describe what is FTD in one line what that would be?

**Comment by AbrarNitk** [comment]
### Easy and Fast Programming Language for Building UI

**Comment by gargajit** [comment]
FTD is the front-end language with an easy-to-learn syntax that caters not just to developers but everyone.

**Comment by amitu** [comment]
FTD: programming language for non programmers.

**Comment by amitu** [comment]
FTD: programming language for the next billion programmers
